skyportal/services/test_server/test_server.py

```diff
@@ -2,6 +2,12 @@
 import datetime
```

Comment:
This service name is too generic; test_smtpd or test_email_server or something?

Reply:
This service also listens for test external API calls (the LT follow-up API calls, specifically, for now) on a different port because I was trying to keep all the test servers under one service/file, so I'm not sure about that. Maybe it would better to separate into two distinct services?

Reply:
Oh, right, I see.  Yes, I think the idea is to make one microservice per concept.